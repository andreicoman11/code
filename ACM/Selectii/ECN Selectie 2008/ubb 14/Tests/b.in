5 5
X.XXX
X.X.X
.XXX.
X.XX.
X.X..

5 5
XXX..
..X..
.XXXX
XX...
XX..X

5 5
XXX.X
..X.X
X...X
...XX
..X.X

5 5
XX.X.
XXX.X
..XX.
XX.XX
XX.XX

5 5
.XX.X
.....
..XXX
..X.X
..X..

5 5
.....
.....
.....
.....
.....

5 5
XXXXX
XXXXX
XXXXX
XXXXX
XXXXX

5 5
.....
.....
..X..
.....
.....

5 5
.....
.....
...X.
..XXX
...X.

10 10
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX
XXXXXXXXXX

10 10
X.X.X.X.X.
.X.X.X.X.X
X.X.X.X.X.
.X.X.X.X.X
X.X.X.X.X.
.X.X.X.X.X
X.X.X.X.X.
.X.X.X.X.X
X.X.X.X.X.
.X.X.X.X.X

10 10
..........
..........
..........
..........
..........
..X.......
..........
..........
..........
..........

10 10
..........
..........
..........
..........
..........
..........
..........
..X.......
.X........
..........

1 2
XX

1 3
X.X

1 3
.X.

3 1
.
X
.

3 1
X
.
X

1 1
X

1 1
.

16 16
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X
X.X.X.X.X.X.X.X.
.X.X.X.X.X.X.X.X

2 2
X.
X.

2 2
.X
X.

2 2
.X
XX

2 2
.X
XX

2 2
XX
.X

3 3
.X.
.X.
.XX

3 3
X..
X.X
XXX

3 3
XX.
X.X
...

3 3
..X
XXX
XX.

3 3
..X
..X
...

3 3
XXX
.X.
X..

1 2
..

1 3
XXX

1 4
..XX

1 5
.XXXX

1 6
.X.X.X

1 7
.X..XXX

1 8
.XX.XX..

1 9
X..XXXX.X

1 10
X.....XXXX

1 11
.XXXX..XXX.

1 12
...X.X..X..X

1 13
XXX..X.X.X.X.

1 14
.X..X.X....X..

1 15
.X.XXXXX.X....X

1 16
XX..XX.XX.....X.

2 2
XX
..

2 3
X..
...

2 4
.X.X
.X.X

2 5
.XXXX
XX...

2 6
.XXX..
.X.XXX

2 7
.XX....
..X.X.X

2 8
.X.X.X..
X...XXXX

2 9
XXX.XX.X.
XXXXX.XXX

2 10
....X.X.X.
XX..XXXX.X

2 11
X...XX.X.X.
...X.XXXX.X

2 12
X.XXXXXX.X..
XX.X.XX.XX..

2 13
.X...X.X.....
.X.X.X..X...X

2 14
X...XX.X..XXX.
X.XXX.....XX..

2 15
XXX..XXX......X
.X..X.X..X..XXX

2 16
X.X..X.XX.X..XX.
XXXXX..XX..XXX..

3 2
..
.X
.X

3 3
X..
XX.
.XX

3 4
XXX.
.XX.
.X.X

3 5
....X
.X.XX
XXXXX

3 6
X...X.
XX....
.X.XX.

3 7
XX.X...
...XX..
..XX.X.

3 8
.X..XXX.
X.X..XXX
X...X.X.

3 9
.X..X.XXX
.X..XX.XX
.XX.X.XX.

3 10
XX..XXX.XX
....X.X...
...XX..XX.

3 11
XXX.X.XXX..
XX.X...XX.X
XX.XX.X..XX

3 12
...X.X..XX.X
XXXXX..XXXXX
XX..XXX.X..X

3 13
.X..X.......X
XXXXXX....X.X
X..X.X.....X.

3 14
.X.X.....XXXX.
XXX.XX.XX..X.X
.X.X.XX...XX.X

3 15
...X..X.X..X.XX
.X.X...XX.XXXXX
...X.X.X..XX..X

3 16
.XX.X.X.XXXXX.XX
..XXX.XXX.X.....
...XX.XXX.XXX...

2 3
..X
..X

3 3
.X.
..X
.XX

4 3
X..
..X
XX.
.XX

5 3
..X
XX.
.X.
...
...

6 3
XX.
X..
.X.
X.X
.X.
X..

7 3
XX.
XX.
X..
..X
XXX
XX.
X..

8 3
X.X
.XX
..X
XXX
...
..X
.XX
.X.

9 3
.X.
XX.
...
..X
..X
X..
..X
X.X
.X.

10 3
..X
..X
X..
.X.
XX.
...
.XX
.XX
X.X
.X.

11 3
.X.
XXX
.X.
.XX
.X.
..X
.XX
X.X
.XX
XXX
XX.

12 3
.XX
.X.
...
...
.XX
..X
X.X
XX.
X.X
X.X
...
.XX

13 3
X.X
...
.X.
X..
..X
X.X
...
.X.
..X
.XX
..X
.X.
XXX

14 3
X.X
.X.
XX.
..X
X.X
...
.X.
.X.
..X
X..
..X
.X.
XXX
X.X

15 3
..X
.X.
X.X
X.X
.X.
XXX
X..
XX.
XX.
...
.X.
..X
X..
...
X..

16 3
.X.
..X
.XX
...
..X
X.X
X..
X..
..X
.XX
X.X
.X.
...
.X.
XXX
X..

2 2
XX
X.

3 2
..
XX
XX

4 2
.X
XX
XX
X.

5 2
XX
.X
X.
.X
.X

6 2
XX
..
X.
..
.X
.X

7 2
.X
.X
XX
X.
X.
XX
XX

8 2
XX
..
X.
..
X.
..
XX
.X

9 2
.X
X.
.X
XX
.X
XX
XX
..
..

10 2
X.
XX
XX
XX
.X
.X
XX
.X
.X
..

11 2
..
..
..
.X
X.
XX
XX
..
XX
XX
.X

12 2
.X
.X
XX
X.
..
.X
XX
..
.X
X.
XX
XX

13 2
X.
..
XX
X.
XX
.X
XX
..
XX
.X
.X
.X
XX

14 2
..
.X
.X
XX
..
..
XX
XX
..
.X
X.
..
X.
..

15 2
X.
..
XX
XX
..
XX
X.
XX
.X
XX
..
.X
..
X.
X.

16 2
.X
X.
.X
XX
.X
X.
XX
X.
..
XX
..
X.
X.
XX
XX
..

2 1
X
.

3 1
X
X
X

4 1
.
.
X
X

5 1
.
.
.
X
X

6 1
.
.
X
.
.
.

7 1
.
.
.
X
.
X
.

8 1
X
X
X
X
.
.
X
.

9 1
.
X
X
X
X
X
.
.
X

10 1
X
.
.
.
X
X
.
X
.
.

11 1
.
X
X
X
.
.
X
.
.
.
.

12 1
.
.
X
.
X
X
.
.
.
.
X
X

13 1
.
X
X
.
X
X
X
.
X
.
X
.
.

14 1
X
.
.
.
X
X
.
X
X
X
.
X
.
X

15 1
X
.
X
.
X
.
X
.
.
X
.
.
X
.
.

16 1
.
.
X
X
X
X
.
X
X
X
.
.
X
.
.
X

12 15
..X.XXXXX..X...
X...XX....XX...
..XXXXXXXXX.XXX
XXXXX.X.......X
XX..XXXX.XX...X
XX.X.....X..XX.
XX.X...X.X..XXX
.X.XX......XXXX
.XXXX..XXX.X.X.
...XX...XX.X..X
.X..X.XXX.X.XX.
....XXXXX...X.X

9 16
XXXXXXXXXXXX.XX.
XXX.X..X.XX.X...
XX.X..X...X.XX..
....XX..XX...X.X
.X..XX.XX.......
X..X....XX.XX..X
XX...X.X.X.X...X
X..X.X.XX..XXX..
XX..X.X....X...X

15 6
.X..XX
.XX..X
...XXX
......
X.XXXX
X....X
X.XXX.
......
..X...
..X...
X..XX.
..X.X.
.X..XX
.XXXXX
.X...X

3 11
XXXX.X.XX.X
XXXX..X..X.
X..XX..XXX.

3 7
X..X.X.
.XXXX.X
.XX..XX

13 5
.X..X
X.XXX
X..X.
.XXXX
.X...
.X.XX
X.X.X
X..XX
.XXX.
.X.X.
....X
.XX..
X..XX

9 12
..XXXXXX.XX.
X...X.X.X.XX
.X...X.X..XX
..X..XX....X
XX.XXXX...X.
.....X.XX...
XXXX....X...
X.X.X.XXX..X
XX.X...X....

1 5
.X...

9 7
XXX.X..
.X..X.X
XX.XXXX
XXXXXX.
X...XXX
.XX.XX.
X.XXX..
XX...X.
XX.X...

10 13
.X.X.X..XX.XX
X.XX.X.X.X.X.
XXX..XXXXXXX.
.X.X..XX.XX..
.X.XXXX..XXXX
..X.XX.XX.X..
.X..X.XX.XX..
.XXX..XX.XX..
....X..X...XX
.X..XXXXXXX..

1 8
XXXX..X.

15 3
XXX
.XX
.X.
X.X
X.X
XX.
XX.
..X
XX.
X..
...
..X
..X
.X.
...

5 1
.
X
.
.
.

3 10
.X.X..XX.X
....X.X...
X..XX.XXXX

13 12
.X.....XXXX.
X.XXXXXX....
..X.XX.X....
X.XX.XX.XXXX
XXX.XX...XX.
XX..XXXX.X.X
XX..X.XXXX..
XX.XXX.X.X.X
X.X..X.X..XX
X......X..XX
X.XX....X...
.X.XX..X.XXX
.....X.X...X

15 5
XX.XX
XXX.X
X.XXX
.XX.X
..X.X
.XX..
XX..X
XXXXX
X.X.X
...X.
X..XX
XX..X
.XX.X
.X...
XX..X

13 5
...X.
XXXX.
.XXXX
X.X..
XX.X.
..XXX
.X...
.X..X
.....
X....
.XX.X
X.XXX
XX..X

11 12
.XXXX.XX.XX.
XX...XX.XX.X
XXXX....X..X
.X...X...X..
..XX....X...
.X..XX....X.
X.XX.XXX.XX.
X.X.X.XX.XXX
X.X.X..XXXX.
..X..X....XX
XXXXX..X....

9 9
..X.X.X.X
.X.X..X.X
.X..X...X
.XXX.X...
XXX.....X
X.XXX.XX.
...XXXX.X
X.XX.X..X
..X.XXX..

11 8
..XXX..X
.X.XX.XX
XXXX.X..
.XXX.XXX
.XXXX.XX
...XX...
XXXXX.XX
XXXX.XXX
X.XXXX..
.XX...XX
XX.X.XXX

10 15
.X.X.XX..X...X.
XXXXX..X..XX.XX
..XX.....XXXXX.
X.X..XXX.XX.X..
XX..XX.X.X.X.XX
.XXXX..X.XX...X
.X..XX..XXXXXX.
X...XX..X.XXXX.
XXX....X...X...
.X.X..XX...XX.X

6 9
XXX.XX...
XX.X...X.
...X.X.X.
XXXXX...X
.X..XX...
XXX..XXXX

11 12
XXXX....X.X.
XX......X...
XX......XX.X
XX.X..X.XXXX
X.........XX
XX...X.X...X
.XX.X.XXXXX.
X.X.XX.XX..X
.X.XX.X...X.
.X....XXX..X
..XX...XX..X

15 14
.X..XXXXXX...X
.XX.XX...XXXXX
..XXX....XXXXX
X.XX.XX.X.XXX.
.XXX..X.X..XXX
..XX.....X.X.X
X.X..X.X.XX...
X.......XX..XX
.......XX..XXX
X..X.X.XX.XX..
.X.XXX.X...XX.
XXX....X..X..X
....X.X..X.X.X
..X...XXX.X.XX
...X..XX......

9 8
..XXXXXX
X.XXX.X.
XXX...XX
...X.X..
.XX....X
.XX.XXXX
.X..XXXX
XX....X.
X..XX...

13 8
...X....
..X...XX
XX.XXX.X
........
X.XXXXXX
X.XX.X..
XX.XX.X.
....X.X.
.XX...XX
.XX...XX
.X.X..X.
XXXX.XX.
X.X...X.

12 9
XX...X.XX
XXX..X..X
XXXXXX.X.
XX.X..XXX
.X.....X.
.X.X.X..X
.X.X.XXXX
X....XX.X
X...X.XXX
...X..XX.
.X.X.XX.X
X..X...XX

15 5
X...X
X.X..
X.X.X
X..X.
X...X
.....
..XX.
XXX.X
X.XXX
....X
X..XX
.X.XX
X....
.XXXX
XXXXX

12 15
..XXX....XXX...
X..X.....X..X.X
.X.X.....X.X.X.
XX..XXXX..XXX..
..X.X........XX
X.X.XX.X.X....X
.XX.XX.X..X...X
.XXX.XX...XXXXX
X...XX...XXXXXX
XX.X...X.XX....
.XX.X.XX.X...XX
XXX.X.XXXX.X.X.

9 2
XX
..
X.
.X
X.
.X
XX
XX
..

14 10
XX..XXX..X
XXX.X.XXXX
XXX..XX...
..X.XX.X..
X.XX.X.X.X
....XX.XXX
XX...XXXXX
XXX..X.X..
..XX.X..XX
.XXXXXXX..
.X.XX..XXX
..XXX.X.XX
..X..X....
X.X.X.XX..

13 9
XX....X.X
.X.X.XX.X
.XX.XXXXX
X.XX.X.XX
X..X.X..X
..XXXXX.X
XXX...XX.
X..X..X..
.XXXXXXX.
XX.X...XX
...XX..XX
XXX.X...X
XX.XXXX.X

4 16
X...X.X.X..X...X
.....X...X..XX.X
X.X.XX...XX.....
X...X.X.X....X..

16 14
XX..XXXX..X..X
.X..XXX.X.XX..
X.X....X...XX.
X.XX.X...XX..X
XXX.X.......X.
XXX.X..XX.....
XX...X.XXXXX..
X..XXX...XX...
XXX...X..XXX.X
XXXXX.XX..XXX.
XXXX.X....XXX.
.X.XXX.X.XXXXX
X.X.X.XXX..XXX
..X.......X..X
XX.XX.XX.XXX.X
XXXXXXXXXX.XXX

10 7
.XX.X..
....XX.
XXXX..X
.X.X.X.
X.XXX.X
.X.XXX.
...X..X
.X....X
XXX.XX.
.XX.X.X

12 12
XXXX..XXX..X
.X.X.XXXXX..
...XX...XX.X
X.XXX...XXXX
..X..XXX..XX
...X.X...X..
.X..XX...XXX
XXXX...X..X.
X.XX..X..X..
.XX.X.XX.X.X
.XXXXXXX....
XX.XXX.XX.XX

13 2
.X
..
X.
XX
..
XX
.X
..
.X
.X
..
XX
X.

14 11
XX.X.X.X.XX
X...XX.X.X.
..XX.X.X..X
X.XXX.XXX.X
X..X.X.XXXX
XX...XXXX..
..X.....XX.
X..X.X.XX.X
.X.XX..X.X.
XXXX.....X.
XX.XXXXX...
XX.X..XX.XX
.X.X.....XX
..XX.XX.X.X

15 1
.
.
X
.
X
.
.
.
.
.
X
.
.
X
X

16 9
.XX......
.XXX.X...
.X...X.X.
X.XX..XX.
..XX.XX.X
X.X..X.X.
XX...XX..
..XX...XX
X....XX.X
XXX.XX...
..XX...X.
XX..XXXXX
XX.X..X.X
XX.....XX
X...XXX..
X.XXX.XX.

3 1
.
X
.

8 6
XXXXXX
..X.X.
.....X
X.....
XX.X.X
.X.X.X
...X..
.X.X..

12 2
X.
XX
XX
XX
X.
X.
.X
.X
..
..
.X
.X

3 10
...X.....X
.XXXX..XXX
...X.XXX..

3 12
XXX....XX..X
XX...X.X...X
X.XXX.....XX

6 16
..XX.X..XXXXXXXX
X..X...XX..X...X
XXXXX.......X..X
.X.XX.XXX...X.X.
XX.XX...X.X....X
X.XX..X.X..X.X..

7 8
.XXX.XX.
..X..XXX
...X...X
.XX..X.X
...X..X.
XXXXXXXX
....XXX.

9 11
XX..XX.X...
X..X.X....X
X.XXXX..X..
.XXXXX.XXX.
XXX.X....XX
XXXX..XX...
XX.XX.XX..X
X..XXXXX...
X....XX.X.X

13 14
XX..XXX...X..X
.XX.X....X..XX
X.X......XX..X
XX...XX..X...X
..XXX.X.X.X..X
.XXX..XXXX.X.X
X.X.XXXXXX.X..
X.X....X.XX.XX
..XX..X...X.X.
......XXXXXXXX
XX.XXX.XX.X...
XX.XXXXXX...XX
XXXXX...XX.XXX

7 9
XX.XX.XX.
X..X.....
X..X.X..X
..XXXX..X
X.XXXX...
..X.XX.XX
X....X..X

9 4
.XX.
.XXX
X.XX
XX..
XX.X
X.X.
.X.X
.XXX
...X

6 14
X.....XX.XXXX.
X..XX..XX.X.X.
...XX....X.X.X
XXXXXX..X.X.XX
...X.X.X...X.X
X...X..X..XX.X

14 14
.X.XX..XXXX.X.
XXXX.....X..X.
X.X.XXXX..X.XX
XX..XXXXXX..XX
XX..X....XX.X.
XX..X..XXXX...
XXXXXXXXXX.XX.
.XX.XX...X..XX
.XX..X...X...X
.XXXX.XX...XX.
XX.XXXXXXX..XX
XX..XX.XX.X.XX
...X.X..X.XX..
......X....XXX

1 8
XX..XXX.

2 2
XX
..

8 9
XX..X.X..
XXX..X.XX
..XX..X..
......XX.
.XX......
XXXXXX...
XXX....X.
XXX...XX.

11 13
XX.X.XX...X..
X..X..X.X.X..
....X....X.X.
XX..X.X.X...X
..XX.X..XXX.X
XX..X......XX
...XXXX.X.XXX
XX.X..X....XX
X.XX..XX..XXX
..XX.XXXXX.XX
XX.XX...X..XX

7 1
.
X
.
X
.
X
X

3 10
.XXX.XXX..
.X..XXX.XX
....XXX.XX

16 12
XX..X..X.XX.
.X..X..X...X
XX.X...XX...
XXXXX.X.X...
X.X.X...X...
......XXX..X
X..XX..X....
....X.X.....
XX.X.X....XX
...X.X..X.XX
XX.X.XX.XX.X
..XX...X.XX.
X....X.X.X..
XXXX.XX..XX.
.X.X.X..XXX.
X..X.X..XXXX

7 2
XX
XX
XX
X.
X.
XX
..

4 14
X..X.X..XX....
XXXX.XXX.X.X.X
.XXX...X..X.X.
X..XXXXX.X.X.X

10 3
X.X
XX.
.XX
X..
..X
.XX
.XX
...
.X.
.X.

10 9
.X.X.XXXX
.XX..X.XX
......X.X
.XX.XX.X.
X..X.X.X.
XX.XX...X
...X.X..X
.XX.....X
.XX.X..X.
XXX.XXX.X

5 15
....XX.X...X.XX
XX.X...X.XX.X..
.XX.XXX..XX..XX
XXX.X..X.X.XXXX
....XX.X.X.....

11 14
.XXXX.X.XX.X.X
.X.X.X.XX..X..
..XXX.X..XXX.X
XXX..X...XX..X
X.X....XXXXX..
XXXX...XXX.X.X
.XX..XX.X.....
X.X.XXX.XX...X
..XXXX.XXX...X
...X..XX...XXX
.X......XXX.X.

5 4
X...
X..X
XXX.
...X
...X

11 12
XXXXXXXXX..X
X...X.....X.
.XXX.XX..XXX
X.X.XXX.X...
XX.X..X.XX.X
X.XX....XX..
....X.X..XXX
XX..X.X...X.
...X..XXXX.X
X.X....XXXXX
.XXXX..X.X.X

10 10
X.XX..X..X
..XX.X.XXX
.XXXX.X..X
.X.XXXX.X.
....X.XX.X
..XX.XXXXX
.....XXX..
X.XX..X.X.
XX.X..XXXX
..XX..X..X

9 5
.XXXX
.X...
X..X.
.X.X.
....X
XX.X.
X.XXX
.XX.X
.XXXX

8 2
XX
XX
X.
.X
XX
X.
..
.X

5 7
XXX...X
.X..X..
X.XXX..
X.XX.XX
..X.XXX

8 11
..X..XX.X.X
X.X..X...XX
.XXXX.XXX.X
XXX.XXX.X.X
X.XXX..X.X.
.X.XX.X...X
X......XXX.
XX.........

10 7
XXX..XX
X.X.XXX
X.XXXXX
.XXXX..
X......
X..X.XX
...X.X.
.XXXX..
...X...
....X..

16 3
.XX
XX.
X.X
XX.
XX.
XX.
XX.
XXX
.X.
...
XXX
XX.
XX.
X..
XX.
.XX

16 4
.XXX
X..X
XXXX
X.XX
XX.X
..XX
X.X.
....
.X..
XX..
X.XX
..XX
X...
...X
.XXX
X..X

7 11
X.X........
X.X.X...XX.
......X..XX
.XXX..X.XXX
XXX.X...XX.
..XX.XXXX.X
X....XXXXX.

4 10
XX.X..X..X
XXX..X..XX
XX.X..XXXX
.X....X...

14 2
..
X.
XX
..
X.
.X
X.
.X
XX
.X
..
X.
XX
X.

10 6
..X.X.
...XXX
..XX..
.X..X.
XX.X.X
..XXXX
....X.
..X...
....X.
X..X.X

2 2
..
.X

3 2
XX
.X
.X

15 9
.....XX.X
.XXX.X.X.
..X...X.X
X..X.XX..
.XXXX..XX
..XX....X
..X.XXXXX
.X...X...
.XXX.XXXX
..XXX...X
XXXXXX.X.
XX.XXXXXX
X.XX.XX.X
....X.X..
...X..XXX

10 6
X.X...
X.....
.X...X
.X.X..
XX...X
.....X
..XXXX
....XX
..XX.X
...X.X

3 7
.X...XX
XX.XX.X
X...X..

12 1
.
.
X
X
X
.
.
X
.
.
.
X

8 16
.X.X....XX..XXXX
X.X.XX..X....XX.
.....X..X.XX.XX.
.XXXXX.X.X.X.X.X
XX.XX.X.XX.X....
.X.XXXX..XXX..X.
XX.XXX.XX..X.X..
X..X...XX.X.X.X.

2 16
.XXXX..X....X...
X.X..X.X....XX.X

14 11
.XX..X.X..X
.X.XX.X...X
X..X.XXXX..
.XXXX..XXXX
..X.XXXX..X
..XXX....XX
..XXX..X.X.
...X.X.X.X.
XX.....X.XX
X..X.XXX...
XXX.X..X.X.
X.X.X......
XX..XX..XX.
.X..X.....X

12 8
...XXX.X
XXXX...X
..X.X.XX
...X.XXX
.X.XXXX.
.X...XXX
X...XXX.
XX..X.X.
XX.XXX.X
.XXXXX..
XX...XX.
.XX.X.X.

1 14
XX.X.....X..X.

4 8
X.X.X..X
X..X...X
X...XXXX
.X.X.X..

5 11
X.....XXXX.
.X....XXXX.
..X..X.X..X
XXX.X...XX.
.XX.X.X.XXX

15 2
..
XX
.X
..
..
X.
.X
..
..
.X
X.
.X
XX
X.
..

7 10
X...XX....
X.XX.XX...
XXX.X.X...
X..X..XXX.
.XX.X..XXX
XX..XXXXXX
XX.XXX.XX.

2 4
.X..
....

11 16
...XX.X....XXXX.
.....XXXX..XXXXX
.....XX.XXXX....
.X...X.X..XX....
XXX..X.X.....X.X
X..XXXXXX..XXX.X
X..X.X.XX.X.XXXX
.XXXX....X.....X
XX..X..X...X....
..X.X..XXX..XX..
XXXX.XXX..XXXX..

3 14
X..XX.X....XXX
X.XX.X......XX
X.X.X.....X..X

2 9
X.XX...X.
XXX..X..X

5 14
...XXX....XX.X
...X...X....X.
...XX.XX.X.X..
.XXXX.X....XX.
..XXX.X.X.X.XX

0 0
