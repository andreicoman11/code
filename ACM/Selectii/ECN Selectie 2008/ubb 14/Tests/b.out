You have to tap 6 tiles.
You have to tap 12 tiles.
You have to tap 7 tiles.
You have to tap 13 tiles.
Damaged billboard.
You have to tap 0 tiles.
You have to tap 15 tiles.
You have to tap 11 tiles.
You have to tap 1 tiles.
You have to tap 44 tiles.
You have to tap 38 tiles.
You have to tap 41 tiles.
You have to tap 54 tiles.
You have to tap 1 tiles.
You have to tap 2 tiles.
You have to tap 3 tiles.
You have to tap 3 tiles.
You have to tap 2 tiles.
You have to tap 1 tiles.
You have to tap 0 tiles.
Damaged billboard.
You have to tap 2 tiles.
You have to tap 2 tiles.
You have to tap 1 tiles.
You have to tap 1 tiles.
You have to tap 1 tiles.
You have to tap 4 tiles.
You have to tap 5 tiles.
You have to tap 3 tiles.
You have to tap 5 tiles.
You have to tap 5 tiles.
You have to tap 4 tiles.
You have to tap 0 tiles.
You have to tap 1 tiles.
You have to tap 1 tiles.
Damaged billboard.
You have to tap 3 tiles.
You have to tap 3 tiles.
You have to tap 2 tiles.
You have to tap 7 tiles.
You have to tap 5 tiles.
Damaged billboard.
You have to tap 8 tiles.
You have to tap 6 tiles.
Damaged billboard.
You have to tap 9 tiles.
You have to tap 9 tiles.
You have to tap 2 tiles.
Damaged billboard.
You have to tap 2 tiles.
Damaged billboard.
You have to tap 6 tiles.
You have to tap 4 tiles.
You have to tap 6 tiles.
You have to tap 6 tiles.
You have to tap 9 tiles.
Damaged billboard.
You have to tap 18 tiles.
You have to tap 10 tiles.
You have to tap 14 tiles.
Damaged billboard.
You have to tap 18 tiles.
Damaged billboard.
You have to tap 3 tiles.
You have to tap 7 tiles.
Damaged billboard.
You have to tap 9 tiles.
You have to tap 14 tiles.
You have to tap 8 tiles.
You have to tap 15 tiles.
You have to tap 17 tiles.
You have to tap 9 tiles.
You have to tap 18 tiles.
You have to tap 24 tiles.
Damaged billboard.
You have to tap 24 tiles.
You have to tap 23 tiles.
Damaged billboard.
You have to tap 3 tiles.
You have to tap 7 tiles.
You have to tap 2 tiles.
You have to tap 12 tiles.
You have to tap 10 tiles.
Damaged billboard.
You have to tap 10 tiles.
You have to tap 22 tiles.
Damaged billboard.
You have to tap 17 tiles.
You have to tap 19 tiles.
Damaged billboard.
You have to tap 27 tiles.
You have to tap 21 tiles.
You have to tap 1 tiles.
Damaged billboard.
You have to tap 4 tiles.
You have to tap 3 tiles.
You have to tap 6 tiles.
You have to tap 5 tiles.
You have to tap 5 tiles.
Damaged billboard.
You have to tap 12 tiles.
Damaged billboard.
You have to tap 13 tiles.
You have to tap 13 tiles.
You have to tap 15 tiles.
Damaged billboard.
You have to tap 17 tiles.
Damaged billboard.
You have to tap 1 tiles.
You have to tap 1 tiles.
You have to tap 1 tiles.
You have to tap 2 tiles.
You have to tap 3 tiles.
You have to tap 3 tiles.
You have to tap 8 tiles.
You have to tap 5 tiles.
Damaged billboard.
You have to tap 6 tiles.
You have to tap 5 tiles.
Damaged billboard.
You have to tap 7 tiles.
You have to tap 6 tiles.
You have to tap 85 tiles.
You have to tap 74 tiles.
You have to tap 46 tiles.
Damaged billboard.
You have to tap 13 tiles.
Damaged billboard.
You have to tap 52 tiles.
Damaged billboard.
You have to tap 30 tiles.
You have to tap 69 tiles.
You have to tap 3 tiles.
You have to tap 21 tiles.
Damaged billboard.
You have to tap 16 tiles.
You have to tap 81 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 64 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 66 tiles.
You have to tap 28 tiles.
You have to tap 64 tiles.
Damaged billboard.
Damaged billboard.
Damaged billboard.
You have to tap 49 tiles.
Damaged billboard.
You have to tap 84 tiles.
You have to tap 9 tiles.
You have to tap 83 tiles.
You have to tap 61 tiles.
You have to tap 34 tiles.
Damaged billboard.
You have to tap 32 tiles.
You have to tap 60 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 7 tiles.
You have to tap 72 tiles.
You have to tap 3 tiles.
Damaged billboard.
You have to tap 9 tiles.
You have to tap 17 tiles.
You have to tap 12 tiles.
You have to tap 48 tiles.
Damaged billboard.
You have to tap 51 tiles.
Damaged billboard.
You have to tap 35 tiles.
Damaged billboard.
You have to tap 42 tiles.
Damaged billboard.
You have to tap 2 tiles.
You have to tap 2 tiles.
Damaged billboard.
You have to tap 75 tiles.
You have to tap 4 tiles.
You have to tap 14 tiles.
You have to tap 97 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 16 tiles.
You have to tap 52 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 10 tiles.
You have to tap 62 tiles.
You have to tap 39 tiles.
You have to tap 21 tiles.
You have to tap 8 tiles.
You have to tap 13 tiles.
Damaged billboard.
You have to tap 34 tiles.
You have to tap 25 tiles.
You have to tap 40 tiles.
Damaged billboard.
You have to tap 19 tiles.
You have to tap 13 tiles.
You have to tap 29 tiles.
You have to tap 3 tiles.
Damaged billboard.
You have to tap 57 tiles.
You have to tap 31 tiles.
You have to tap 10 tiles.
You have to tap 4 tiles.
You have to tap 72 tiles.
You have to tap 11 tiles.
Damaged billboard.
You have to tap 52 tiles.
Damaged billboard.
You have to tap 18 tiles.
Damaged billboard.
Damaged billboard.
You have to tap 37 tiles.
You have to tap 5 tiles.
You have to tap 106 tiles.
Damaged billboard.
Damaged billboard.
Damaged billboard.
